src/context.cpp

```diff
@@ -379,6 +379,27 @@ namespace librealsense
     }
 
 #ifdef BUILD_WITH_DDS
+    struct sid_index
+    {
+        int16_t sid;   // Stream ID; assigned based on the stream TYPE, really
+        int8_t index;  // Used to distinguish similar streams like IR L / R, 0 otherwise
+
+        sid_index( int sid_, int index_ )
+            : sid( static_cast< int16_t >( sid_ ) )
```

Comment:
Why cast and not save as int? If it is received from LibRS as int then there is no point casting it. If need to cast when sending over DDS cast there, If the message interface will change, no need to update here

Reply:
It's a copy from `dds_stream_uid` that was removed. You're right, but I don't think it's important right now?

Reply:
I prefer that we will fix this now, we have to many things we commented on as "for later" and we'll forget to update them all

Reply:
fixed